src/current/v24.1/migration-overview.md

```diff
@@ -310,35 +312,32 @@ Using this method, consistency is achieved by only performing the cutover once a
 The following is a high-level overview of the migration steps. For considerations and details about the pros and cons of this approach, see [Migration Strategy: Lift and Shift]({% link {{ page.version.version }}/migration-strategy-lift-and-shift.md %}).
 
 1. Stop application traffic to your source database. **This begins downtime.**
-1. Move data in one of the following ways:
-	- {% include {{ page.version.version }}/migration/load-data-import-into.md %}
-	- {% include {{ page.version.version }}/migration/load-data-third-party.md %}
-	- {% include {{ page.version.version }}/migration/load-data-copy-from.md %}
-1. After the data is migrated, you can use [MOLT Verify]({% link {{ page.version.version }}/molt-verify.md %}) to validate the consistency of the data between the source database and CockroachDB.
+1. Use [MOLT Fetch]({% link {{ page.version.version }}/molt-fetch.md %}) to move the source data to CockroachDB.
+1. After the data is migrated, use [MOLT Verify]({% link {{ page.version.version }}/molt-verify.md %}) to validate the consistency of the data between the source database and CockroachDB.
 1. Perform a [cutover](#cutover-strategy) by resuming application traffic, now to CockroachDB.
 {% comment %}1. If you want the ability to [roll back](#all-at-once-rollback) the migration, replicate data back to the source database.{% endcomment %}
 
 ### Zero Downtime
```

Comment:
This is a different claim from earlier, when we only talked about "minimizing" or "reducing" downtime.